kubernetes/src/main/kotlin/com/google/kubernetes/tools/settings/KubernetesSettingsConfigurable.kt

```diff
@@ -46,6 +60,35 @@ class KubernetesSettingsConfigurable : Configurable {
                         false /*chooseJarsAsFiles*/,
                         false /*chooseJarContents*/,
                         false /*chooseMultiple*/))
+
+        skaffoldBrowser.textField.document.addDocumentListener(object : DocumentListener {
+            val skaffoldExecutorService = object : SkaffoldExecutorService() {
+                override var skaffoldExecutablePath: Path = Paths.get(skaffoldBrowser.text)
+            }
+
+            override fun changedUpdate(event: DocumentEvent?) {
+                checkSkaffold()
+            }
+
+            override fun insertUpdate(event: DocumentEvent?) {
+                checkSkaffold()
```

Comment:
What happens if you type in the field? Is it fast enough - it seems to call skaffold on each type?

Reply:
yeah. I saw no perceptible slow down in my tests.